spec/src/legacy_app_spec.js

```diff
@@ -20,4 +20,12 @@ describe('LegacyApp', () => {
       expect(app.switchTo).toHaveBeenCalledWith('main', data.user);
     });
   });
+
+  describe('#logClosedApp', () => {
+    it('is called when the app is exiting', function() {
+      spyOn(app, 'logClosedApp');
```

Comment:
Not sure we should rely on the implementation of `LegacyApp` here, since that code will be modified by the app developer. Could we write a mocked app that includes a `willDestroy` handler spy instead? That way the tests will still pass even after the developer changes the implementation of LegacyApp. Obviously, the developer could also choose to delete all these specs.